tools/sgen/input/FluentUIStyle.yml

```diff
@@ -445,6 +444,19 @@ AP_MSFListCellTokens:
   sublabelFont: $Typography.caption1
   trailingItemForegroundColor: $Colors.Foreground.neutral3
   trailingItemSize: $Icon.size.medium
+  
+AP_MSFHeaderFooterTokens:
+  backgroundColor: [ default: $Colors.Background.neutral1, divider: $Colors.Background.neutral5 ]
+  headerHeight: [ default: $Spacing.xxxlarge, divider: $Spacing.xlarge ]
```

Comment:
Where is the **divider: $Spacing.xlarge**  being used? (same for the background color)

Reply:
Not using right now but there is a header style called divider, which will be implemented later

Reply:
As per our offline conversation, we decided not to add stuff that the code is not using right now.